frontend/src/components/common/StudentDetailCommon.vue

```diff
@@ -574,6 +588,28 @@ export default {
     formatPen,
     formatMincode,
     formatDob,
+    boldFormatter: function (char){
+      if(/[A-Z]/.test(char)){
+        this.diff = '𝗔'.codePointAt (0) - 'A'.codePointAt (0);
+      }else{
+        this.diff = '𝗮'.codePointAt (0) - 'a'.codePointAt (0);
+      }
+      return String.fromCodePoint(char.codePointAt(0)+this.diff);
+    },
+    copyInfo: function(){
+      this.dobCopy=formatDob(this.studentCopy.dob,'uuuu-MM-dd','uuuu/MM/dd');
+      this.copyTxt=
+          'PEN: '.replace(/[A-Za-z]/g,this.boldFormatter)+(this.studentCopy.pen==null?'':this.studentCopy.pen)+'\n'+
```

Comment:
I think it would be better to use `(this.studentCopy.legalLastName ||  '' ` instead of `(this.studentCopy.legalLastName == null ? '' : this.studentCopy.legalLastName)`, the code would be more concise.